tests/dummy/app/routes/application.ts

```diff
@@ -18,6 +20,17 @@ export default class ApplicationRoute extends Route {
     this.setupTracking();
   }
 
+  @action
+  didTransition(): void {
+    if (
+      config.environment !== 'test' &&
```

Comment:
Any specific reason for not doing this in tests? Is this interfering with something?

Reply:
tbh I stole this from the docfy documenation page 🥇 so I can't really tell you why this is in - I guess this is here because we don't want to scroll the window when in test env - as the app runs inside the testing container in the test-env.